src/App.js

```diff
@@ -1,11 +1,89 @@
 import React, { Component } from 'react';
 import './App.css';
+import axios from 'axios';
+import MovieList from './MovieList';
+import SearchBar from './SearchBar';
+
 
 class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      searchedMovies: [],
+      savedMovies: [],
+      searchText: ''
+    };
+  }
+  handleSearchTextChange(event) {
+    this.setState({
+      searchText: event.target.value
+    });
+  }
+
+  getSavedMovies(){
+    axios.get('http://localhost:4000/movies')
+      .then(resp => {
+        this.setState({
+          savedMovies: resp.data
+        })
+      })
+  }
+  handleSubmit(event) {
+    event.preventDefault();
+    axios.get(`https://api.themoviedb.org/3/search/movie?api_key=2ab4bec4c09a675c9baa72dfe29d2ab6&language=en-US&query=${this.state.searchText}&page=1&include_adult=false`)
+      .then(resp => {
+        this.setState({
+          searchedMovies: resp.data.results
+        });
+      });
+  }
+  addMovie(event, movieName){
+    let id;
```

Comment:
I think we really want to move this type of logic away from `addMovie` and maybe more into the component that lets us add a movie or not.

`addMovie` should really only be concerned with adding movies, not determining if it can be added or not.